STYLE_GUIDE.md

```diff
@@ -0,0 +1,720 @@
+| **What is isn't** | Chatty, Hyperbolic, Cloying, Insincere | Stuffy, Dictatorial, Smug, Unsure, Untested | Wordy, Lengthy, Verbose, Rambling | Stilted, Pompous, Chummy, Pedantic | Vague, Wandering, Confusing, Ambiguous, Surprising, Indecisive, Redundant | Insulting, Condescending, Insensitive, Indifferent | Fancy, Esoteric, Perplexing, Unintelligible, Complicated, Unusual |  Pedantic, Invalid, Inaccurate, Varying, Disorganized, Careless, Sloppy |  Infallible, Evasive, Devious, Obfuscating |
+
+
+
+## Blog posts
+
+Blog posts provide an informal approach to educating or inspiring readers through the personal perspective of the authors. Brief posts generally accompany service or feature releases, and longer posts may note best practices or provide creative solutions. Each post must provide a clear community benefit.
+
+To enhance the strengths of the blogging platform, follow these post guidelines:
+
+**Be conversational and informal.**
+
+Posts tend to be more personable, unlike technical documentation. Ask questions, include relevant anecdotes, add recommendations, and generally try to make the post as approachable as possible. However, be careful of slang, jargon, and phrases that a global audience might not understand.
+
+**Keep it short.**
+
+Deep topics don’t necessarily require long posts. Shorter, more focused posts are easier for readers to digest. Consider breaking a long post into a series, which can also encourage repeat visitors to the blog channel.
+
+**Avoid redundancy.**
+
+Posts should add to the conversation. Instead of repeating content that is already available elsewhere, link to detail pages and technical documentation. Keep only the information that is specific to the post solution or recommendations.
+
+**Connect with other content.**
+
+All posts should contain one or more calls to action that give readers the opportunity to create resources, learn more about services or features, or connect with other community members. Posts should also include metadata tags such as services, solutions, or learning levels to help readers navigate to related content.
+
+## Inclusive content
+
+OpenSearch content strives to be inclusive and free of bias. We use inclusive language to connect with the diverse and global OpenSearch audience. This means we are careful in our word choices. Inclusive and bias-free content improves clarity and accessibility of our content for all audiences. We avoid ableist and sexist language and language that perpetuates racist structures or stereotypes.
```

Comment:
Please add the following content after this paragraph:

### Offensive terms (Heading 3)

Don't use the following offensive terms in OpenSearch content.

[Table (I will provide separately)]

### Sensitive terms (Heading 3)

Avoid using these terms in OpenSearch content in the form described in the following table.

[Table (I will provide separately)]